docs/getting-started/faq.mdx

```diff
@@ -0,0 +1,133 @@
+# Frequently Asked Questions
+
+Common questions about Zephyr Cloud and how to get the most out of the platform.
+
+:::details{title="How does Zephyr speed up development?"}
+
+Zephyr integrates with [bundlers](/bundlers/webpack) like Webpack, Rspack, and Vite as a side-effect of your build process to handle deployment automatically. Instead of configuring complex deployment pipelines, Zephyr deploys your application to the edge network as part of your standard build.
+
+**Key benefits:**
+
+- Instant deployments (sub-second)
+- No separate deployment configuration needed
+- Automatic version management
+- Built-in rollback capabilities
```

Comment:
Not sure if this is a big highlight to be honest?
Talk about rollback without talking about environments feels too off.

Maybe a better highlight is the ability to BYOC.